packages/template-retail-react-app/package.json

```diff
@@ -27,6 +28,7 @@
         "card-validator": "^8.1.0",
         "chalk": "1.1.3",
         "commerce-sdk-isomorphic": "^1.5.2",
+        "commerce-sdk-react": "^2.3.0-dev",
```

Comment:
Note the current version in `develop` after the 2.3.0 release is now [2.4.0-dev](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/commerce-sdk-react/package.json#L3).

Reply:
🤔 We're working off the long-lived feature branch, which does not have the latest from `develop` yet. So I'll need to keep it up to date.

Reply:
Ok, I've updated the PR with the latest from `develop` branch.